sycl/test/reduction/reduction_usm.cpp

```diff
@@ -0,0 +1,83 @@
+// UNSUPPORTED: cuda
+// Reductions use work-group builtins not yet supported by CUDA.
+
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
+// RUNx: env SYCL_DEVICE_TYPE=HOST %t.out
```

Comment:
Why do we have RUNx here.  If host is not suppoerted add corresponding feature to UNSUPPORTED section with proper comment.

Reply:
I see many other 'RUNx' in other SYCL LIT tests, that is why I thought it is Ok to have it.

Vladimir, Can you please give one good example where it is done properly (with UNSUPPORTED keyword)
Did you really mean to write something like 'UNSUPPORTED: host', or just delete the line with RUNx and write a comment section. Something like:
// TODO: enable this test on HOST when HOST gets support of intel::reduce() and barrier() features.

Reply:
@vladimirlaz I did additional changes (4th commit in this PR) per your comment.